integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/proxyservice/test/loggingProxy/ProxyServiceEndPointThroughURLTestCase.java

```diff
@@ -52,18 +52,15 @@ public void testLoggingProxy() throws Exception {
         String symbol = response.getFirstElement().
                 getFirstChildWithName(new QName("http://services.samples/xsd", "symbol")).getText();
         assertEquals(symbol, "WSO2", "Fault: value 'symbol' mismatched");
-    }
 
-    @Test(groups = "wso2.esb", description = "VerifyLogs")
-    public void testVerifyLogs() throws Exception {
         LogViewerClient logViewerClient = new LogViewerClient(context.getContextUrls().getBackEndUrl(),
                 getSessionCookie());
         assertNotNull(logViewerClient.getRemoteLogs("INFO", "getQuote", "", ""),
-                      "Request INFO log entry not found");
-        assertNotNull(logViewerClient.getRemoteLogs("INFO", "<ns:symbol>WSO2</ns:symbol>", "", ""),
-                      "Request INFO log entry not found");
-        assertNotNull(logViewerClient.getRemoteLogs("INFO", "ns:getQuoteResponse", "", ""),
-                      "Request INFO log entry not found");
+                "Request INFO log entry not found");
+        assertNotNull(logViewerClient.getRemoteLogs("INFO", "<ns:symbol>WSO2</ns:symbol>", "",
+                ""), "Request INFO log entry not found");
```

Comment:
Why did we move the `""),` part down? Better to keep it in the previous line for readability.